src/ui/js/Game.js

```diff
@@ -1680,53 +1680,102 @@ Game.pageAddGameNavigationFooter = function() {
 
 // Display a footer-style message with the list of skills in this game
 Game.pageAddSkillListFooter = function() {
-  var gameSkillDiv = $('<div>', {
+  var hasButtonSpecial = false;
+  var hasDieSkill = false;
+  var isFirstButtonSpecial = true;
+  var isFirstDieSkill = true;
+  var isFirstInteraction;
+  var skillDesc;
+
+  var buttonSpecialDiv = $('<div>', {
+    'text': 'Button specials in this game: ',
+  });
+  var dieSkillDiv = $('<div>', {
     'text': 'Skills in this game: ',
   });
 
-  var firstSkill = true;
-  var firstInteract;
-  var skillDesc;
   $.each(Api.game.gameSkillsInfo, function(skill, info) {
     skillDesc = skill;
+
     if (info.code) {
+      // add a die skill ...
+      skill += ' (' + info.code + ')';
       skillDesc += ' (' + info.code + ')';
-    }
-    skillDesc += ': ' + info.description;
+      skillDesc += ': ' + info.description;
+
+      // ... with interactions
+      isFirstInteraction = true;
+      $.each(info.interacts, function(otherSkill, interactDesc) {
+        if (isFirstInteraction) {
+          skillDesc += '<br/><br/>Interaction with other skills in this game:';
+          isFirstInteraction = false;
+        }
+        skillDesc += '<br/> * ' + otherSkill + ': ' + interactDesc;
+      });
 
-    firstInteract = true;
-    $.each(info.interacts, function(otherSkill, interactDesc) {
-      if (firstInteract) {
-        skillDesc += '\n\nInteraction with other skills in this game:';
+      if (isFirstDieSkill) {
+        hasDieSkill = true;
+        isFirstDieSkill = false;
+      } else {
+        dieSkillDiv.append('&nbsp;&nbsp;');
```

Comment:
Similarly, here, you can collect all of your descriptions into an array of descriptions, and then use `.join('&nbsp;&nbsp;')` to combine them.  See the assembly of `capturedDieText` in `Game.gamePlayerStatus()` for an example.